main.js

```diff
@@ -1 +1,80 @@
-console.log('Hello world')
\ No newline at end of file
+{console.log('Hello world')
+let time, seconds, fullSec, fullMin, fullHour, clicked, turned;
+const frame = 60;
+
+const DOMS = {
+    startBtn: "#btn-start",
+    hours: "#hours",
+    minutes: "#minutes",
+    seconds: "#seconds",
+}
+
+const formatNumber = (number, time) =>

```

Comment:
Comment here! Try adding comments to all your functions the [JSDoc](https://jsdoc.app/tags-returns.html) way